.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,85 @@
+name: .NET CI
+
+on:
+  pull_request:
+    branches: [ master, main ]
+    paths-ignore:
+    - '**.md'
+    - '.github/**'
+
+jobs:
+  build:
+    name: Build and test Linux
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    # Install all supported .NET versions
+    - name: Install .NET Core 3.1
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.x
```

Comment:
I'd say it's time to remove support for .NET Core 3.1, support ended 2022-12-13. We've got .NET 6 targeting in 6.0.0. If 3.1 gets in the way, submit a separate PR for it.

Reply:
I don't have an objection if .NET Framework was removed but .NET Standard 2.0 was kept. That means users that still haven't migrated have an easier path by keeping support for .NET Framework 4.6.1+ (really .NET Framework 4.7.2) and means .NET Framework only technologies/APIs can be removed. Maybe there is no need for .NET Standard either, however check out some open source libraries that depend on Windsor to find out their targets.

Can you create an issue describing exactly what you propose and ask for others to provide their opinions/proposals.

Reply:
For now I switched the netcore3.1 test to net 6 and got the net462 tests running as well. I will create a separate issue for discussing end of support for older frameworks.